io_scene_nif/modules/animation/animation_export.py

```diff
@@ -63,31 +63,78 @@ def get_flags_from_extend(self, cyclic):
             return 4 # 0b100
 
     
-    def export_keyframes(self, arm, bone, parent_block):
-        print("export_keyframes",bone.name)
-        action = arm.animation_data.action
-        if not action:
-            return
-        if bone.name in action.groups:
-            action_group = action.groups[bone.name]
+    def export_keyframes(self, b_obj, bone, parent_block):
```

Comment:
Should use default values of None, will save on the caller.

Reply:
Done!